parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -381,26 +382,7 @@ try
 
         Write-Log "Write kubelet startfile with pod CIDR of $podCIDR"

```

Comment:
Is this comment accurate?
Doesn't the pod CIDR get saved to config in all the powershell in kubeletstart.ps1?

Reply:
removed comment